forge/routes/index.js

```diff
@@ -4,5 +4,8 @@ module.exports = fp(async function (app, opts, done) {
     app.addHook('preHandler', app.checkAuth)
     await app.register(require('./files'), { prefix: '/v1/files', logLevel: app.config.logging.http })
     await app.register(require('./quota'), { prefix: '/v1/quota', logLevel: app.config.logging.http })
+    if (app.config.context) {
```

Comment:
```suggestion
    if (app.config.context?.type) {
```

We should probably check type is at least something?